bin/update_cucumber

```diff
@@ -1,11 +1,6 @@
 #!/usr/bin/env php
 <?php
 
-if (PHP_VERSION_ID < 70400) {
```

Comment:
This check is no longer needed since this package can only be installed on PHP 8.

Additionally, [PHPStan understands this and shows a warning](https://github.com/phpstan/phpstan/issues/12528).